src/modules/mavlink/mavlink_receiver.cpp

```diff
@@ -1448,7 +1448,7 @@ MavlinkReceiver::handle_message_set_attitude_target(mavlink_message_t *msg)
 					}
 
 					if (!_offboard_control_mode.ignore_thrust) { // dont't overwrite thrust if it's invalid
-						att_sp.thrust = set_attitude_target.thrust;
+						att_sp.thrust_x = set_attitude_target.thrust;
```

Comment:
Is x here correct? Further down it's using z.

Reply:
Fixed, for now use thrust_z as offboard is used for multicopters